orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/conditions/ConditionConfigurationProperties.java

```diff
@@ -78,4 +79,12 @@ public void setClusters(List<String> clusters) {
   public void setActiveConditions(List<String> activeConditions) {
     this.activeConditions = activeConditions;
   }
+
+  public boolean isSkipWait() {
+    return configService.getConfig(Boolean.class, "tasks.evaluateCondition.skipWait", skipWait);
+  }
+
+  public void setSkipWait(boolean skipWait) {
+    this.skipWait = skipWait;
```

Comment:
if the only use of this is for testing, I would prefer this style:

```
// remove the skipWait field and setSkipWait method
  public boolean isSkipWait() {
    return configService.getConfig(Boolean.class, "tasks.evaluateCondition.skipWait", false);
  }

// in the test, mock the response with
1 * conditionsConfigurationProperties.isSkipWait() >> true
```

Reply:
Reasonable